nginx_config_reloader/__init__.py

```diff
@@ -79,9 +79,12 @@ def process_IN_MOVE_SELF(self, event):
         raise ListenTargetTerminated
 
     def handle_event(self, event):
-        if self.notifier and self.notifier.check_events(timeout=100):
-            self.logger.info("Detected other reloads in queue, skipping this one.")
-            return
+        try:
```

Comment:
Is the idea here to empty the event queue? Took me a couple looks to get this. Maybe put it in a separate function, so here we just read sth like `self.empty_event_queue()` except with a better fn name. 

Also why not `self.notifier._eventq.clear()`? It's a deque, right?